packages/upload/src/ajax.ts

```diff
@@ -0,0 +1,91 @@
+import type {
+  ElUploadProgressEvent,
+  ElUploadRequestOptions,
+  ElUploadAjaxError,
+} from './upload'
+
+function getError(action: string, option: ElUploadRequestOptions, xhr: XMLHttpRequest) {
```

Comment:
```suggestion
function getError(action: string, option: ElUploadRequestOptions, xhr: XMLHttpRequest) : string | ElUploadAjaxError {
```
Should it be like that ? that's the first time I'm doing TypeScript 😛

Reply:
No need to explicit annotate return type for internal function